tests/behat/fixtures/d10/config/sync/core.entity_form_display.node.page.default.yml

```
@@ -34,6 +40,30 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  field_date_only:
+    type: datetime_default
+    weight: 122
+    region: content
+    settings: {  }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)